### PR TITLE
Core 1513 thanks page ftds

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Message for FTDs added to thanks page
- Settings added to kiva admin to manage this feature

ftds_desktop_cut

ftds_mobile_cut

### CODE REVIEW

File: src/graphql/query/thanksPage.graphql [hunk 1]
```diff
@@ -132,6 +132,14 @@ query checkoutReceipt($checkoutId: Int!, $visitorId: String) {
 			key
 			value
 		}
+		ftd_message_enable: uiConfigSetting(key: "thanks_page_ftd_message_enable") {
```
Comment (dyersituations) [inline src/graphql/query/thanksPage.graphql:135]: Is there a reason why were using a config instead of an experiment?

Comment (roger-in-kiva) [reply src/graphql/query/thanksPage.graphql:135]: This is not a test, but we want the ability to turn this off

File: src/components/Thanks/ShareStepper.vue [hunk 2]
```diff
@@ -17,6 +17,10 @@
 				<span v-if="commentsMode">
 					<span class="data-hj-suppress">{{ lenderName }}</span>, go the extra mile
 				</span>
+				<span v-else-if="isFirstLoan">
+					<!-- eslint-disable-next-line max-len -->
+					<span class="data-hj-suppress">{{ lenderName }}</span>, You’re support means the world to <span class="data-hj-suppress">{{ loanName }}</span>. Let us help you change another life.
```
Comment (dyersituations) [inline src/components/Thanks/ShareStepper.vue:22]: Does `loanName` need to be suppressed?

Comment (roger-in-kiva) [reply src/components/Thanks/ShareStepper.vue:22]: Commonly this is the borrower name, should be?

Comment (dyersituations) [reply src/components/Thanks/ShareStepper.vue:22]: It does look like we've hidden the loan name elswhere.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**